packages/serverless/src/apollo/app/group.ts

```diff
@@ -147,7 +147,9 @@ const groupListByUser = publicWrapper<
   TGroupListByUserRequest,
   TGroupListByUserResponse
 >(JOI_GROUP_LIST_USER, async (_root, { databaseName, userName }) =>
-  new ModelUserGroup(databaseName).listGroupByUserName(userName)
+  (
+    await new ModelUserGroup(databaseName).listGroupByUserQuery({ userName })
```

Comment:
We want the client to able to get group list by user filter, for example the request should be:

```
{
   databaseName: string,
   userQuery: Partial<{
       userName: string,
       publicKey: string,
       email: string
   }>
}
```

Reply:
fixed